pylint/extensions/kwargs.py

```diff
@@ -0,0 +1,89 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/pylint/blob/main/CONTRIBUTORS.txt
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from astroid import BoundMethod, UnboundMethod, nodes
+
+from pylint.checkers import BaseChecker
+from pylint.checkers.utils import safe_infer
+from pylint.interfaces import INFERENCE
+
+if TYPE_CHECKING:
+    from pylint.lint import PyLinter
+
+
+class KeywordChecker(BaseChecker):
+    """Checks related to keyword arguments."""
+
+    name = "kwargs"
+    msgs = {
+        "W3501": (
+            "Call to `%s` missing keyword argument `%s`.",
+            "consider-using-keyword-argument",
+            "Used when method is called without specifying a keyword argument.",
```

Comment:
```suggestion
            "When using a literal directly in a function call, it's very hard to know what the argument is"
            " if a positional argument is used because there's no variable name or attribute name to "
            "rely on when reading. Using a keyword argument there's at least the attribute name to "
            "help the reader understanding the call."
```